### ISSUE TITLE
GitHub screenshot

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/lesson-example#69]
On #67, @timtomch uses his own user for the screenshots. Can we use @swcarpentry user for the screenshots? This will enable more people to take the screenshots and avoid to change the text each time that someone update the files based on their screenshots.

**Comment by timtomch** [comment]
Makes sense. However, it does limit who can update the lesson to those who have access to that account...
